src/index.html

```diff
@@ -10,7 +10,44 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page">
+    <div class="search-bar">
+      <form
+        action="/"
+        class="search-bar__form search-bar__form--big"
+        method="post"
+        data-qa="big"
+      >
+        <img class="icon icon--big" src="./images/Search.svg" alt="Search img">
+        <label>
```

Comment:
why do you need this label without any attributes? I think you can remove it, and from small input too